showcase.json

```diff
@@ -3711,5 +3711,27 @@
     "stats": "1000+ Monthly Users",
     "display": "Normal",
     "logo": "/img/showcase/vaultbet_logo.webp"
+  },
+  {
+    "id": "icto",
+    "name": "ICTO",
+    "oneLiner": "ICTO is a one-stop platform automating token operations on the Internet Computer.",
+    "tags": [
+      "Tools / Infrastructure",
+      "NFT",
+      "DAO",
+      "DeFi"
+    ],
+    "description": "ICTO is an automated token management platform that simplifies vesting schedules, recurring payroll distributions, token lock and tokenized crowdfunding for projects building on the Internet Computer. It provides modular smart contracts, allowing any project to transparently handle token operations without technical friction.",
+    "usesInternetIdentity": true,
+    "website": "https://icto.app",
```

Comment:
The website does not work for me.

Reply:
Thanks, Our website is working now.

Reply:
Hey @FernNguyen - I don't think we can publish this for now. The website is not ready for this. You should have at least a proper landing page that explains the project.

Reply:
Oh sorry, I just wanted to show our project on dfinity grants page to inform to let everyone know about this, because it's true that our project is in the construction phase. I will try again after fully satisfying the project introduction information.

Reply:
Hey @domwoe, We've just updated the Landing page on https://icto.app . Please take a moment to check it out.

Reply:
@FernNguyen I've seen it. Great progress! I've already provided some feedback on Slack.